platform/solr/solr-dependencies/pom.xml

```diff
@@ -168,7 +168,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.9.5</version>
+            <version>${jackson.version}</version>
```

Comment:
We weren't using this property before?

Reply:
It's being used in one other place in DDF.